modules/http/src/main/scala/higherkindness/mu/http/package.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package higherkindness.mu
+import scala.annotation.StaticAnnotation
+
+package object http {
```

Comment:
Why do we need a package object here instead of a standard one?

Reply:
Done at 
https://github.com/higherkindness/mu/pull/581/commits/b8be4b74924844d2ecddcb626dd2477db816207c